### ISSUE TITLE
warning about unknown props

### ISSUE THREAD

**Issue by beckend** [tomchentw/react-toastr#63]
By just mounting the container on react 15.2.0:

```javascript
Warning: Unknown props `toastMessageFactory`, `toastType`, `preventDuplicates`, `newestOnTop` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by ToastContainer)
    in ToastContainer (created by Toastr)
    in Toastr (created by App)
```

**Comment by mikecousins** [comment]
Yeah, the React team says this needs to be fixed as it will break in the next version: https://github.com/facebook/react/issues/7152#issuecomment-230273479

**Comment by RangarajKaushikSundar** [comment]
Seems like there is a bit of refactoring in place. Good share @beckend @mikecousins !

**Comment by Harryandrew** [comment]
Did anyone find the cause and a subsequent fix for this?

**Comment by RangarajKaushikSundar** [comment]
@Harryandrew The component will work on lower versions of React. I have included this component in my project and am using React v15.1.0. I don't seem to face this issue, the code would need refactoring to circumvent this problem.

**Comment by tomchentw** [comment]
Released [v2.8.0](https://github.com/tomchentw/react-toastr/releases/tag/v2.8.0)
